stx-genesis/src/lib.rs

```diff
@@ -3,65 +3,57 @@ use std::io::{self, BufReader};
 
 use libflate::deflate;
 
-use stacks::{
-    burnchains::{bitcoin::address::BitcoinAddress, Address},
-    chainstate::stacks::db::{AccountBalance, VestingSchedule},
-    chainstate::stacks::StacksAddress,
-};
+pub struct GenesisAccountBalance {
+    /// A STX or BTC address (BTC addresses should be converted to STX when used).
+    pub address: String,
+    /// Balance in microSTX.
+    pub amount: u64,
+}
+
+pub struct GenesisAccountVesting {
```

Comment:
Same here -- we should avoid calling it `vesting`.

Reply:
Done!